OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -448,12 +450,7 @@ public bool CanEnterTargetNow(Actor self, Target target)
 			if (target.Positions.Any(p => self.World.ActorMap.GetActorsAt(self.World.Map.CellContaining(p)).Any(a => a != self && a != target.Actor)))
 				return false;
 
-			var res = target.Actor.TraitOrDefault<Reservable>();
-			if (res == null)
-				return true;
-
-			UnReserve();
-			Reservation = res.Reserve(target.Actor, self, this);
+			MakeReservation(target.Actor);
```

Comment:
OK I double checked this myself and I believe this to be a sensible change - anything calling this is expecting the aircraft to move to this target - so I think changing the behaviour so the aircraft releases any old reservation is fine.